src/read_csv.py

```diff
@@ -5,32 +5,38 @@
 import csv
 import sys
 import pandas as pd
+import os
 
+PATH = sys.path[0]
 
 def transform_csv_from_bad_format_to_better_format(data_list, new_csv_name):
     """
     Takes in dataList, which represents the original CSV, and writes a new
     CSV file that has mapped all the point with their name.
     """
     # Find the first empty row and create a dictionary to map the points to their names:
-    split = 0
+    split = 1
     point_to_name = {}
-    while split < len(data_list) and data_list[split][0] != "":
+    while "Point_" in data_list[split][0]:
         point_to_name[(data_list[split][0])[:-1]] = data_list[split][1]
         split += 1
-    
+
     # Write the new data to a CSV:
     with open(new_csv_name, 'w') as f:
         writer = csv.writer(f)
         
         # Construct and write the first line
         first_line = [data_list[split + 1][0], data_list[split + 1][1]]
-        for j in range(2, len(data_list[split + 1]), 1):
-            first_line.append(point_to_name[data_list[split + 1][j]])
         writer.writerow(first_line)
-        
+        writer.writerow("")
+
+        first_line = ["Date","Time"]
+        for key, value in point_to_name.items():
+            first_line.append(value)
+        writer.writerow(first_line)
+
         # Write the rest of the lines:
-        for i in range(split + 2, len(data_list), 1):
+        for i in range(split + 5, len(data_list), 1):
             writer.writerow(data_list[i])
 
 
```

Comment:
Do we use this get_dataframe_from_csv or is it an example? if its just an example I'd like to remove it (sorry, I think i added it, but want to check if removing it is ok with yall)

Reply:
I believe that its just an example that you and maybe Zephyr worked on early on. Fine by me to remove